Make the readme a bit clearer (#1243)
playmint/ds: tutorial/README.md

# A guide to building your own game with Downstream

## Welcome to the learning zone!

This page is your one stop shop for learning how to build on top of Downstream. We recommend that you follow the Tutorials to get up and running. The tutorials will explain how to set up your work space and then run you through small example Islands that focus on a small set of mechanics. 

> - Once you feel you are up to speed on what is currently possible on Downstream, you can check out our Existing Maps and Games!

Reviewer: a list with only one thing probably doesn't deserve to be a list :)

```suggestion
Once you feel you are up to speed on what is currently possible on Downstream, you can check out our Existing Maps and Games!
```

> - The Existing Maps are various things that Playmind has created in Downstream. There are some smaller examples like the Gate Demo and some larger examples like the quest-map.

Reviewer 2: ```suggestion
The Existing Maps are various things that Playmint has created in Downstream. There are some smaller examples like the Gate Demo and some larger examples like the quest-map.
```
Playmind -> Playmint

> The Existing Games are full games that have been developed by external developers and one by Playmint! They will give you an insight into how far Downstream has already been pushed!

Reviewer: look like you intended this to be a linebreak, but didn't give it the extra space? 

```suggestion

The Existing Games are full games that have been developed by external developers and one by Playmint! They will give you an insight into how far Downstream has already been pushed!
```

> For more in depth information, check out the References page!

Reviewer: Not sure it's really "more in depth" just different

and should it be a link?

```suggestion
See the [References](./REFERENCE.md) for information on common tooling and commands.
```

or something like that
